### PR TITLE
Remove redundant author doc comments

### PR DESCRIPTION
**Pull request by IAmActuallyCthulhu**
Removed the following author doc comment from files, since it the information is present in copyright disclaimer at the top of file and git history.

```
/**
	@author Juan Linietsky <reduzio@gmail.com>
*/
```

This was done in relation to my other PR #31261.

### CODE REVIEW

File: core/io/image_loader.h [hunk 1]
```diff
@@ -37,10 +37,6 @@
 #include "core/os/file_access.h"
 #include "core/ustring.h"
 
-/**
-	@author Juan Linietsky <reduzio@gmail.com>
-*/
-
 /**
  * @class ImageScanLineLoader
  * @author Juan Linietsky <reduzio@gmail.com>
```
Comment (aaronfranke) [inline core/io/image_loader.h]: I think you missed a spot.

**Review by aaronfranke — APPROVED** [review]
Seems like a good idea to me. Git history is great for keeping track of authors, and it also does a much better job anyway.

File: core/io/image_loader.h (hunk 1 above)
Comment (akien-mga) [reply core/io/image_loader.h]: Needs rebase now that #31269 is merged.

Comment (IAmActuallyCthulhu) [reply core/io/image_loader.h]: I have done the rebase and force pushed it.

**Comment by akien-mga** [comment]
For the reference, @reduz is fine with it:
```
11:44 <Akien> reduz: OK with removing your @author line from old core files? #31266
11:44 <IssueBot> #31266: Remove redundant author doc comments | https://git.io/fj7oP
13:37 <reduz> Akien: yes sure
```
(And nice timestamp :))

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**